samples/drivers/led_strip/boards/stamp_c3.overlay

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024 Croxel, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/led/led.h>
+
+#include <zephyr/dt-bindings/led/led.h>
+
+&uart0 {
+	status = "okay";
+	current-speed = <2500000>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+	data-bits = <7>;
```

Comment:
Don't we need to remove the parity bit too ?

Reply:
Yes, we dont need to use parity bit here. Just 7 data bits along 1 start and 1 stop bits is enough to represent 3 bits of rgb data.